supporting_content/dimensions/region.md

```diff
@@ -0,0 +1,88 @@
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column            |
+|----------|--------------------------|-------------------|
+| AWS      | CUR                      | Product_region    |
+| GCP      | Big Query Billing Export | Location.location |
+| Azure    | EA, MCA                  | ResourceLocation  |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | Example Value                                                                   |
+|----------|--------------------------|---------------------------------------------------------------------------------|
+| AWS      | CUR                      | us-east-1 | eu-west-1 | us-gov-east-1                                          |
```

Comment:
Use commas instead of pipes. When you use pipes, they won't get rendered with the output.
```suggestion
| AWS      | CUR                      | us-east-1, eu-west-1, us-gov-east-1                                          |
```
(Same for the other lines)